### ISSUE TITLE
Add phpstan 

### ISSUE THREAD

**Issue by BackEndTea** [symfony/demo#1260]
A lot has changed since https://github.com/symfony/demo/pull/733, we now have baselines and a good way to ignore errors.

Is there any interest in adding it to the project?

**Comment by javiereguiluz** [comment]
Yes ... this demo project follows Symfony practices as close as possible ... so, given that Symfony code already uses PHPStan, we can try to add it here too. Thanks!

**Comment by burned42** [comment]
I gave this a try and created https://github.com/symfony/demo/pull/1265 I hope that's ok :)
